### PR TITLE
Grammatical, Capitalization, and Articulation corrections on the README 

### PR DESCRIPTION
**Pull request by DGKSK8LIFE**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,64 +14,62 @@ That's it.
...
+LARBS is a script that automatically installs and automatically configures a fully-functioning, minimal terminal-and-vim-based Arch Linux environment.
 
-LARBS was originally intended to be run on a fresh install of Arch Linux, and
+LARBS was originally intended to be run on a fresh installation of Arch Linux, and
 provides you with a fully configured diving-board for work or more
-customization. But LARBS also works on already configured systems *and* other
-Arch-based distros such as Manjaro and Parabola (although Parabola,
-which uses slightly different repositories might miss one or two minor
+customization. But, LARBS also works on preconfigured systems *and* other
+Arch-based distrobutions such as Manjaro and Parabola (although Parabola,
```
Comment (frilox042) [inline README.md:22]:
```suggestion
Arch-based distributions such as Manjaro and Parabola (although Parabola,
```

Comment (DGKSK8LIFE) [reply README.md:22]: oh shit; sorry man

**Comment by DGKSK8LIFE** [comment]
ty @frilox042

**Comment by DGKSK8LIFE** [comment]
why did you close this?

**Pull request closed without merging**